docs/data-ingestion.asciidoc

```diff
@@ -13,12 +13,17 @@ This section explains how to adapt data ingestion according to your needs.
 [[tune-apm-server]]
 == Tune APM Server
 
+Version 6.5 of the APM Server introduced a new <<intake-api,intake API>>. If you have not upgraded your APM Agent, you'll continue to use the deprecated v1 intake API endpoint. If you have upgraded your agent, you'll use the new v2 intake API endpoint. Understanding which endpoint your APM Agents use to communicate with your APM Server is vital to correctly tuning your data ingestion. Read the <<intake-api-changes-65, intake API changes>> documentation to learn more.
```

Comment:
in v2 there is a `max_event_size` added, is that documented already somewhere else, will be done in a future PR, or wont be done? I don't think users will need to tune that, so not opposing to not document it altogether.

If you document this setting, it should be in connection to an apm-server error "event too large"

Reply:
Yes! It's currently documented in the [config options](https://www.elastic.co/guide/en/apm/server/current/configuration-process.html#configuration-v2-api), and in the Upgrading to 6.5 [server config changes](https://www.elastic.co/guide/en/apm/server/current/upgrading-to-65.html#server-config-changes-65). 

I've linked to these sections with a new "common problem". Is event too large a `413` error?

Reply:
just checked, and event too large is a 400.

Reply:
Thanks @jalvz - I've updated the status code